cni-plugins/sriov-passthrough-cni/plugin/sriov-passthrough-cni

```diff
@@ -23,64 +22,30 @@ function main() {
     esac
 }
 
-function flip_vfs_to_default_drivers() {
-    for file in $(find /sys/devices/ -name *sriov_totalvfs*); do
-        pfroot=$(dirname $file)
-
-        # flip all available VFs to default driver
-        echo 0 > $pfroot/sriov_numvfs
-        cat $file > $pfroot/sriov_numvfs
-    done
-}
-
-function from_cni_args() {
-    local var_name="${1:?}"
-    sed -nE "s/.+?${var_name}=([^;]*?)(;.+?|$)/\1/p" <<< "$CNI_ARGS"
-}
-
 function setup_netns() {
     mkdir -p /var/run/netns
     ln -sfT "$CNI_NETNS" "$CONTAINER_NETNS_LINK"
 }
 
-function setns_sriov_ifs() {
-  local sriov_vfs=( /sys/class/net/*/device/virtfn* )
-  local ifs_arr ifs_name
-  for vf in "${sriov_vfs[@]}"; do
-    ifs_arr=( "$vf"/net/* )
-    for ifs in "${ifs_arr[@]}"; do
-        ifs_name="${ifs%%\/net\/*}"
-        ifs_name="${ifs##*\/}"
-        log "Adding vf: $ifs_name to netns $CNI_CONTAINERID"
-        ip link set "$ifs_name" netns "$CNI_CONTAINERID"
-    done
-  done
+function setns_sriov_pfs() {
+  local pf_array=()
+  local -r sriov_pfs=( $(find /sys/class/net/*/device/sriov_numvfs) )
+  [ "${#sriov_pfs[@]}" -eq 0 ] && log "FATAL: No sriov PFs found" && exit 1
```

Comment:
Same here let's add a if statement

Reply:
lets see what the result of https://github.com/kubevirt/project-infra/pull/884#discussion_r565317084
and do accordingly